Pod/Classes/SEGAdobeIntegration.m

```diff
@@ -13,6 +13,137 @@
 #import "ADBMediaHeartbeat.h"
 
 
+@implementation VideoAnalyticsProvider
+/**
+ * The system time in millis at which the playhead is first set or updated. The playhead is
+ * first set upon instantiation of the PlaybackDelegate. The value is updated whenever
+ * updatePlayheadPosition is invoked.
+ */
+long initialTime;
+/** The current playhead position in seconds. */
+long playheadPosition;
+/** The position of the playhead in seconds when the video was paused. */
+long pausedPlayheadPosition;
+/** The system time in millis at which {@link #pausePlayhead} was invoked. */
+long pausedStartedTime;
+/**
+ * The updated playhead position - this variable is assigned to the value a customer passes as
+ * properties.seekPosition in a "Video Playback Seek Completed" event or properties.position in
+ * a "Video Content Started" event
+ */
+long updatedPlayheadPosition;
+/** The total time in seconds a video has been in a paused state during a video session. */
+long offset = 0;
+/** Whether the video playhead is in a paused state. */
+BOOL isPaused = false;
+
+
+/**
+  Sets initial time upon delegate creation. This will be called
+  on Video Playback Started, at the time the VideoAnalyticsProvider
+  instance is created.
+ */
+- (void)startDelegateWithInitialTime
```

Comment:
Open to suggestions on what to name this — I know start implies that this initializes the delegate, so I can change this to `setInitialTime`